tests/cypress/e2e/features/shortcuts.js

```diff
@@ -0,0 +1,244 @@
+        const searchItemClass = searchDescription ? '.cvat-shortcuts-settings-item-title' : '.cvat-shortcuts-settings-item-description';
+        const randomSearchItem = searchDescription ? 'random description shortcut' : 'random title shortcut';
+        cy.get('.cvat-shortcuts-settings-search input').focus();
+        for (const searchShortcut of searchShortcuts) {
+            cy.get('.cvat-shortcuts-settings-search input').type(searchDescription ? searchShortcut.description : searchShortcut.title);
+            cy.get(searchItemClass).should('have.length', 1);
+            cy.get(searchItemClass).contains(searchDescription ? searchShortcut.title : searchShortcut.description);
+            cy.get('.cvat-shortcuts-settings-search input').clear();
+        }
+        cy.get('.cvat-shortcuts-settings-search input').type(randomSearchItem);
+        cy.get(searchItemClass).should('not.exist');
+        cy.get('.cvat-shortcuts-settings-search input').clear();
+        cy.get('.cvat-shortcuts-settings-search input').blur();
+    }
+
+    function registerF2F3(shouldExist) {
+        cy.get('.ant-list-item').should('exist').and('be.visible');
+        cy.get('.ant-list-item .ant-select').first().click();
+        cy.realPress(['F2']);
+        cy.wait(200);
+        cy.realPress(['F3']);
+        cy.get('.ant-modal-content').contains('Conflicting shortcuts detected');
+        cy.get(
+            shouldExist ?
+                '.ant-modal-content .ant-modal-confirm-btns .ant-btn-primary' :
+                '.ant-modal-content .ant-modal-confirm-btns .ant-btn-default',
+        ).click();
+        cy.get('.ant-list-item .ant-select').first().within(() => {
+            cy.get('.ant-select-selection-overflow-item').contains('f2 f3').should(shouldExist ? 'exist' : 'not.exist');
+        });
+    }
+
+    function saveSettings() {
+        cy.get('.cvat-settings-modal .ant-modal-footer .ant-btn-primary').click();
+    }
+
+    function checkShortcutsMounted(label) {
+        cy.get('.cvat-shortcuts-modal-window-table').should('exist').and('be.visible');
+        for (let i = 1; i < 4; i++) {
+            cy.get('.cvat-shortcuts-modal-window-table').contains(label(i));
+        }
+    }
+
+    after(() => {
+        cy.logout();
+    });
+
+    describe('Searching for a shortcut', () => {
+        it('Searching according to description', () => {
+            cy.openSettings();
+            cy.contains('Shortcuts').click();
+            cy.get('.cvat-shortcuts-settings-search input').should('exist').and('be.visible');
+            testSearchShortcuts(true);
+        });
+        it('Searching according to title', () => {
+            testSearchShortcuts(false);
+        });
+    });
+
+    describe('Registration and testing of new shortcuts', () => {
+        it('Registering a new shortcut', () => {
+            cy.get('.cvat-shortcuts-settings-collapse').should('exist').and('be.visible');
+            cy.get('.ant-collapse-header').first().click();
```

Comment:
Please, reduce using `.ant-` classes, it is better to add our own unique `cvat-` classes to different elements